src/legacy/core_plugins/kibana/server/tutorials/apache_logs/index.js

```diff
@@ -22,7 +22,7 @@ import { TUTORIAL_CATEGORY } from '../../../common/tutorials/tutorial_category';
 import { onPremInstructions, cloudInstructions, onPremCloudInstructions } from '../../../common/tutorials/filebeat_instructions';
 
 export function apacheLogsSpecProvider(server, context) {
-  const moduleName = 'apache2';
```

Comment:
@ruflin when I'm fixing the links, I saw this. Should this be `apache` instead of `apache2`?

Reply:
Good catch, this should be apache.